ara/plugins/callback/ara_default.py

```diff
@@ -327,6 +328,27 @@ def __init__(self):
         self.delegation_cache = {}
         self.warned_about_host_length = []
 
+        self.set_handler(signal.SIGINT)
+        self.set_handler(signal.SIGTERM)
+
+    def set_handler(self, sig):
+        original_handler = signal.getsignal(sig)
+
+        def handler(sig, frame):
+            ended = datetime.datetime.now(datetime.timezone.utc).isoformat()
+            status = "failed"
```

Comment:
I tested this and it works, thanks !

I hesitate on whether failed is the appropriate status to use.

The status will be failed but there won't really be a failed task to look at which may be confusing without additional information.

I see two options:
- We re-use the existing ``expired`` status which, like mentioned in the issue, comes from expiring those interrupted or unfinished playbooks that will never finish
- We create a new status, ``interrupted`` to make it clear that this particular playbook was interrupted

Creating a status is not particularly difficult but it is not free either.
We need to adjust the database model, the API, the CLI and the UI accordingly.

Re-using ``expired`` is essentially free and I feel it's a better representation of what happened: the playbook didn't fail but it will also never finish.

This is what it'd look like using expired vs failed:
![Screenshot from 2024-06-30 17-35-11](https://github.com/ansible-community/ara/assets/1291204/f0cc4c3d-5372-4aa5-8510-a1441f20ea71)

What do you think ?

Reply:
I'd like to ship this in a release soon. I will avoid creating a new status for now and use the existing expired one.